<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainMenu</class>
 <widget class="QWidget" name="mainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>562</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI Black</family>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	color : white;
	background-color: #40e649;
	border-radius: 25px;
	height : 50%;
	width:100%;
}
QPushButton:hover{
	color : white;
	background-color: #35bd3c;
	height : 50%;
	width:100%;
}
QWidget {
	background-color:#3d3d3d;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>731</width>
     <height>790</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="bottomMargin">
     <number>300</number>
    </property>
    <item>
     <widget class="QLabel" name="usernameLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { 
	color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="KatalogBtn">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color : white;
	background-color: #40e649;
	border-radius: 25px;
	height : 50%;
	width:100%;
}
QPushButton:hover{
	color : white;
	background-color: #35bd3c;
	height : 50%;
	width:100%;
}</string>
      </property>
      <property name="text">
       <string>Каталог</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SearchBtn">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color : white;
	background-color: #40e649;
	border-radius: 25px;
	height : 50%;
	width:100%;
}
QPushButton:hover{
	color : white;
	background-color: #35bd3c;
	height : 50%;
	width:100%;
}</string>
      </property>
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StatsBtn">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color : white;
	background-color: #40e649;
	border-radius: 25px;
	height : 50%;
	width:100%;
}
QPushButton:hover{
	color : white;
	background-color: #35bd3c;
	height : 50%;
	width:100%;
}</string>
      </property>
      <property name="text">
       <string>Статистика</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="User_ControlBtn">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color : white;
	background-color: #40e649;
	border-radius: 25px;
	height : 50%;
	width:100%;
}
QPushButton:hover{
	color : white;
	background-color: #35bd3c;
	height : 50%;
	width:100%;
}</string>
      </property>
      <property name="text">
       <string>Контроль Пользователей</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ExitBtn">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color : white;
	background-color: #40e649;
	border-radius: 25px;
	height : 50%;
	width:100%;
}
QPushButton:hover{
	color : white;
	background-color: #35bd3c;
	height : 50%;
	width:100%;
}</string>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
